sdks/java/io/cdc/src/test/java/org/apache/beam/io/cdc/DebeziumIOConnectorTest.java

```diff
@@ -118,4 +121,32 @@ public void testDebeziumIODefaultOutputAsJson() {
 
         p.run().waitUntilFinish();
     }
+    
+    @Test
+    public void testDebeziumIOdb2() {
+        PipelineOptions options = PipelineOptionsFactory.create();
+        Pipeline p = Pipeline.create(options);
+        p.apply(DebeziumIO.<String>read().
+                        withConnectorConfiguration(
+                                DebeziumIO.ConnectorConfiguration.create()
+                                        .withUsername("db2inst1")
+                                        .withPassword("=Password!")
+                                        .withConnectorClass(Db2Connector.class)
+                                        .withHostName("127.0.0.1")
+                                        .withPort("50000")
+                                        .withConnectionProperty("database.dbname", "TESTDB")
+                                        .withConnectionProperty("tasks.max", "1")
+                                        .withConnectionProperty("database.hostname","db2server")
+                                        .withConnectionProperty("database.server.name", "db2server")
+                                        .withConnectionProperty("database.cdcschema", "ASNCDC")
+                                        .withConnectionProperty("database.include.list", "TESTDB")
+                                        .withConnectionProperty("table.include.list", "DB2INST1.CUSTOMERS")
+                                        .withConnectionProperty("database.history", FileDatabaseHistory.class.getName())
+                                        .withConnectionProperty("database.history.file.filename", "file2-history.dat")
+                        ).withFormatFunction(new SourceRecordJson.SourceRecordJsonMapper())
```

Comment:
Not necessary. `SourceRecordJsonMapper` is used by default

Reply:
Ok, I will use it for readAsJson().